c_api/include/taichi/taichi_unity.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include <taichi/taichi_core.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+// handle.native_buffer
+typedef struct TixNativeBufferUnity_t *TixNativeBufferUnity;
+
+// function.import_native_runtime
+TI_DLL_EXPORT TiRuntime TI_API_CALL tix_import_native_runtime_unity();
+
+// function.launch_kernel_async
+TI_DLL_EXPORT void TI_API_CALL
+tix_launch_kernel_async_unity(TiRuntime runtime,
```

Comment:
I'm very curious about this set of APIs. Is it fair to say that this treats Unity itself as a new Taichi backend..?

Reply:
Actually not. These APIs' impl merely extract the runtime device from unity's provision; and calls the core APIs (taichi/taichi_core.h). So actually taichi_unity is a user of taichi_c_api.

These async variants only differs from the existing ones by caching all the launch commands in a temporary storage IN GAME THREAD; and actually consume the commands IN RENDER THREAD in the callback function returned by tix_submit_async_unity (note that it returns a void*).

So b